Codes/coursetable.php

```diff
@@ -133,116 +141,115 @@
                             <tr>
                                 <th></th>
                                 <th></th>
-
+                                <?php
+                                $sql="SELECT * FROM controlsheet WHERE course_code='$cd'";
+                                $result = $conn->query($sql);
+                                $row=$result->fetch_assoc();
+                                ?>
```

Comment:
will correct this in testing phase if query is not able to find anything related data in database due to corruption of database for the time being it is okay